packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -250,26 +176,24 @@ export default class Tile extends Surface {
     });
   }
 
+  _getSrc() {
+    return (this.artwork && this.artwork.src) || this._src;
+  }
+
   _imageLoaded() {
     this._Background.alpha = 0; // Since the image is loaded the surface does not need to be shown
   }
 
+  /* ------------------------------ Badge ------------------------------ */
   _updateBadge() {
-    // Remove Badge if no longer required
-    if (!this.badge?.title || this._isCircleLayout) {
-      if (this._Badge) {
-        this._Content.patch({
-          Badge: undefined
```

Comment:
Why was this removed? I noticed that the Label equivalent is still there

Reply:
I can add it back